dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1688,12 +1694,111 @@
       },
       "description": "Represents the confidence scores across all sentiment classes: positive and negative."
     },
+    "EntityList": {
+      "description": "(Optional) describes the entity categories to return",
+      "items": {
+        "type": "string",
+        "x-ms-enum": {
+          "name": "EntityCategory",
+          "modelAsString": true
+        },
+        "enum": [
+          "Default",
+          "All",
+          "Address",
+          "Numeric",
+          "Age",
+          "Currency",
+          "Number",
+          "NumberRange",
+          "Percentage",
+          "Ordinal",
+          "Temperature",
+          "Dimension",
+          "Length",
+          "Weight",
+          "Height",
+          "Speed",
+          "Area",
+          "Volume",
+          "Information",
+          "Temporal",
+          "Date",
+          "Time",
+          "DateTime",
+          "DateRange",
+          "TimeRange",
+          "DateTimeRange",
+          "Duration",
+          "SetTemporal",
+          "Event",
+          "SocialEvent",
+          "CulturalEvent",
+          "NatrualEvent",
+          "Location",
+          "GPE",
```

Comment:
Can you at least add descriptions for each of these - at least those that may not be obvious? See https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-enum.

It could be a different PR, but this will not only help with REST API docs, but generated client libraries as well.